test/core/fixtures/transformation/validation.undeclared-variable-check/undeclared-shorthand-property/exec.js

```diff
@@ -0,0 +1 @@
+({foo})
```

Comment:
Then how do you expect for `t.isReferenced` on `key` to return false, but `t.isReferenced` on `value` to return true, when `key` has the same object identity as `value`?